RELEASE_NOTES.md

```diff
@@ -1,26 +1,13 @@
-# ChefDK 0.16 Release notes
-
-## Chef Generate Improvements.
-* Hyphens in cookbooks are no longer allowed.
-* `chef generate cookbook` now automatically creates files for Chef Automate's workflow features.
- * Files are located in `.delivery` folder in the generated cookbook.
-* `chef generate cookbook` has improved output formatting.
-* `chef generate cookbook` now defaults to creating cookbooks that use Inspec tests by default.
- * Tests are located at `<cookbook>/test/recipes/<recipename>_test.rb`.
- * Tests in existing cookbooks at old location will work fine.
-
-## New `delivery local` command provides a single-command interface to ChefDK.
-* Run `delivery local --help` for details.
-* Customize `delivery local` behavior with the `.delivery/project.toml` file in your project.
-
-## Cookstyle is now default code linter instead of Rubocop.
-* Cookstyle wraps Rubocop, and provides Chef's recommended default set of cops automatically.
-* Execute with `delivery local lint` or `chef exec cookstyle`.
-* If you prefer, you can continue to use Rubocop directly.
-
-## `kitchen-dokken` driver now included in ChefDK.
-
-## Knife updates.
-* `knife cookbook create` and `berks cookbook` commands are deprecated in favor of `chef generate cookbook`.
-* knife supermarket gem has been folded directly into knife, and is no longer a separate gem.
-* `knife cookbook site` command now behaves as a wrapper to `knife supermarket` command.
+# ChefDK 0.19 Release notes
+
+## Inspec
+* InSpec Updated to v1.2.0. See the [InSpec CHANGELOG](https://github.com/chef/inspec/tree/v1.2.0) for details.
+
+## delivery-cli
+* Deprication of Github V1 Backed Projects.
+* Initialization of Github V2 Backed Projects (`delivery init --github`). (Requires Automate Server version `x.x.x` or above)
+* Project name verification with repository name for projects with SCM Integration.
+* New alias `--pipeline` for option `--for`.
+* Honor the custom config on project initialization. (`delivery init -c /my/config.json`)
+* Generate build-cookbook with chefdk build-cookbook command on project initialization.
```

Comment:
This was an enhancement to protect customers to make this mistake: https://github.com/chef/delivery-cli/issues/18